CMakeLists.txt

```diff
@@ -1050,7 +1050,7 @@ endif ()
 
```

Comment:
> i#1557: compatibility update, CMake API dynamorio has changed.

The issue namespaces are different: 1557 is a different issue here.  See prior issues: they use `DRi#1557` to refer to a DR issue in a title.  DynamoRIO/dynamorio/issues/1557 will auto-link in the body.

Reply:
Done